hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/spi/state/impl/RebuiltStateImpl.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app.spi.state.impl;
+
+import com.hedera.node.app.spi.state.StateKey;
+import com.swirlds.fchashmap.FCHashMap;
+import java.time.Instant;
+import java.util.Map;
+import java.util.Objects;
+import javax.annotation.Nonnull;
+
+public class RebuiltStateImpl<K, V> extends StateBase<K, V> {
```

Comment:
Javadoc here? What is the state rebuilt from?

Reply:
This is basically for [aliases FCHashMap](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/services/state/org/StateMetadata.java#L31). It is not currently serialized to ServicesState.

Reply:
👍 thanks!